tracee/event_monitor_ebpf.c

```diff
@@ -758,6 +764,26 @@ static __always_inline int save_argv(submit_buf_t *submit_p, struct pt_regs *ctx
     return 0;
 }
 
+static __always_inline int save_str_arr(submit_buf_t *submit_p, const char __user *const __user *ptr)
```

Comment:
can you suffix the name with `_to_buf` so it's consistent with other similar functions?

Reply:
Will do